packages/vuetify/src/components/VSelect/VSelect.ts

```diff
@@ -419,7 +419,9 @@ export default baseMixins.extend<options>().extend({
 
       icon.children![0].data = mergeData(icon.children![0].data!, {
```

Comment:
This whole thing was a mistake anyway, it should only apply to the default append icon.

Reply:
#10489